common/components/ui/SimpleButton.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React, { Component } from 'react';
+
+const DEFAULT_BUTTON_TYPE = 'primary';
+const DEFAULT_BUTTON_SIZE = 'lg';
+
+const Spinner = () => {
+  return <i className="fa fa-spinner fa-spin fa-fw" />;
+};
+
+type ButtonType =
+  | 'default'
+  | 'primary'
+  | 'success'
+  | 'info'
+  | 'warning'
+  | 'danger';
+type ButtonSize = 'lg' | 'sm' | 'xs';
+
+type Props = {
+  onClick: () => any,
+  text: string,
+  loading?: boolean,
+  disabled?: boolean,
+  loadingText?: string,
+  size?: ButtonSize,
+  type?: ButtonType
+};
+
+export default class SimpleButton extends Component {
+  props: Props;
+
+  computedClass = () => {
+    return `btn btn-${this.props.size || DEFAULT_BUTTON_TYPE} btn-${this.props
+      .type || DEFAULT_BUTTON_SIZE}`;
+  };
```

Comment:
I'd strongly suggest we add [`classnames`](https://www.npmjs.com/package/classnames) to the module stack. It makes a lot of this stuff more manageable, and it's really lightweight.